packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeEthInANutshellModal/StakeEthInANutshellModal.tsx

```diff
@@ -45,27 +56,51 @@ export const StakeEthInANutshellModal = ({ onCancel }: StakeEthInANutshellModalP
     const unstakingPeriod = getUnstakingPeriodInDays(validatorWithdrawTime);
 
     const dispatch = useDispatch();
+    const { elevation } = useElevation();
+
     const proceedToEverstakeModal = () => {
         onCancel();
         dispatch(openModal({ type: 'everstake' }));
     };
 
+    const processes = [
+        {
+            heading: <Translation id="TR_STAKE_STAKING_PROCESS" />,
+            badge: <Translation id="TR_TX_FEE" />,
+            content: <StakingInfo account={account} />,
+        },
+        {
+            heading: <Translation id="TR_STAKE_UNSTAKING_PROCESS" />,
+            badge: (
+                <>
+                    <Translation id="TR_TX_CONFIRMATIONS" values={{ confirmationsCount: 2 }} />{' '}
+                    <Translation id="TR_TX_FEE" />
+                </>
+            ),
+            content: <UnstakingInfo account={account} />,
+        },
+    ];
+
     return (
         <NewModal
-            size="tiny"
             heading={<Translation id="TR_STAKE_STAKING_IN_A_NUTSHELL" />}
+            size="tiny"
             onCancel={onCancel}
             bottomContent={
-                <NewModal.Button onClick={proceedToEverstakeModal}>
-                    <Translation id="TR_GOT_IT" />
+                <NewModal.Button onClick={proceedToEverstakeModal} size="large" isFullWidth>
```

Comment:
Please don't change the size to large and remove the full width prop. I know it looks that way in the task description, but the overall design of modals have changed in the mean time. Sorry about that!